distributed/scheduler.py

```diff
@@ -314,124 +342,110 @@ def _to_dict(self, *, exclude: "Container[str]" = ()) -> dict:
 
 
 class WorkerState:
-    """
-    A simple object holding information about a worker.
-
-    .. attribute:: address: str
-
-       This worker's unique key.  This can be its connected address
-       (such as ``'tcp://127.0.0.1:8891'``) or an alias (such as ``'alice'``).
+    """A simple object holding information about a worker."""
+
+    #: This worker's unique key. This can be its connected address
+    #: (such as ``"tcp://127.0.0.1:8891"``) or an alias (such as ``"alice"``).
+    address: str
 
-    .. attribute:: processing: {TaskState: cost}
+    pid: int
+    name: object
 
-       A dictionary of tasks that have been submitted to this worker.
-       Each task state is associated with the expected cost in seconds
-       of running that task, summing both the task's expected computation
-       time and the expected communication time of its result.
+    #: The number of CPU threads made available on this worker
+    nthreads: int
 
-       If a task is already executing on the worker and the excecution time is
-       twice the learned average TaskGroup duration, this will be set to twice
-       the current executing time. If the task is unknown, the default task
-       duration is used instead of the TaskGroup average.
+    #: Memory available to the worker, in bytes
+    memory_limit: int
 
-       Multiple tasks may be submitted to a worker in advance and the worker
-       will run them eventually, depending on its execution resources
-       (but see :doc:`work-stealing`).
+    local_directory: str
+    services: dict[str, int]
```

Comment:
These attributes were undocumented before and remain it now.
Note that our definition of "public" is "documented in Sphinx". This makes these attributes implicitly private.

There should be a proper pass later on where we document these attributes and decide wether we want to prepend them with _ to hide them from sphinx.